protocol/v2/qbft/spectest/msg_processing_type.go

```diff
@@ -34,7 +33,7 @@ func RunMsgProcessing(t *testing.T, test *spectests.MsgProcessingSpecTest) {
 	ks := spectestingutils.KeySetForCommitteeMember(test.Pre.State.CommitteeMember)
 	signer := spectestingutils.NewOperatorSigner(ks, 1)
 	pre := instance.NewInstance(
-		qbfttesting.TestingConfig(logger, ks, casts.RunnerRoleToConvertRole(msgId.GetRoleType())), // #nosec G104
+		qbfttesting.TestingConfig(logger, ks, spectypes.BeaconRole(msgId.GetRoleType())), // nolint: gosec
```

Comment:
it's strange because it doesn't complain when using `nolint` but does so when I reverted back to using `#nosec`:
https://github.com/ssvlabs/ssv/actions/runs/11877412486/job/33096853887?pr=1845#step:6:27
```
Error: protocol/v2/qbft/spectest/msg_processing_type.go:36:61: G115: integer overflow conversion int32 -> uint64 (gosec)
		qbfttesting.TestingConfig(logger, ks, spectypes.BeaconRole(msgId.GetRoleType())), // #nosec G104
```